content/recipes/prisma.md

```diff
@@ -252,17 +252,22 @@ import { PrismaClient } from '@prisma/client';
 
 @Injectable()
 export class PrismaService extends PrismaClient
-  implements OnModuleInit, OnModuleDestroy {
+  implements OnModuleInit {
+
   async onModuleInit() {
     await this.$connect();
   }
 
-  async onModuleDestroy() {
-    await this.$disconnect();
+  async enableShutdownHooks(app) {
+    this.$on('beforeExit', async () => {
+      await app.close();
+    });    
   }
 }
 ```
 
+> info **Note** The `onModuleInit` is optional - if you leave it out, Prisma will connect lazily on first call to db. We don't bother with `onModuleDestroy`, since Prisma has its own shutdown hooks where it will destroy the connection. For more info on `enableShutdownHooks`, please see [Issues with `enableShutdownHooks`](#issues-with-enableShutdownHooks)
```

Comment:
```suggestion
> info **Note** The `onModuleInit` is optional — if you leave it out, Prisma will connect lazily on its first call to the database. We don't bother with `onModuleDestroy`, since Prisma has its own shutdown hooks where it will destroy the connection. For more info on `enableShutdownHooks`, please see [Issues with `enableShutdownHooks`](#issues-with-enableShutdownHooks)
```